src/patternfly/upgrade-examples/AlertUpgradeExamples/docs/code.md

```diff
@@ -0,0 +1,9 @@
+## Overview
+
+When converting PatternFly 3 components to PatternFly 4 components, you must also take into consideration the layouts and sizings that PatternFly 3 utilized from Bootstrap 3.
+
+## Usage
+
+| PatternFly 3 | Replaced By | PatternFly 4 |
```

Comment:
@matthewcarleton there are a nine elements of alerts that are 1-1 which we can compare. I wrote these in the documentation, and also noted any which are not 1-1 to show that they are not used anymore -- i think this could be helpful. Let me know what you think.

Reply:
ya that works. I wonder if it makes more sense for the docs to look more like the other usage docs where it states the class and where it's used.  Having `<div class="pf-m-success">` might be a bit confusing because it can only be used with `pf-c-alert`. 
so stating the class and then what class or element it can be used with might be more helpful.